test/tchannel_bridge.js

```diff
@@ -89,7 +92,7 @@ describe ('test tchannel span bridge', () => {
             });
 
             // register the server request function.
-            let context = {};
+            let context = new DefaultContext();
             encodedChannel.register(server, 'Echo::echo', context, bridge.tracedHandler(handleServerReq));
```

Comment:
does tchannel use this `context` object as a prototype? A single instance cannot be shared by all inbound requests.